src/igb/Makefile

```diff
@@ -0,0 +1,23 @@
+.ONESHELL:
+SHELL = /bin/bash
+.SHELLFLAGS += -e
+
+MAIN_TARGET = igb.ko
+
+$(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
+	rm -rf ./igb-$(IGB_DRIVER_VERSION)
+	wget -O igb-$(IGB_DRIVER_VERSION).tar.gz "http://downloads.sourceforge.net/project/e1000/igb%20stable/5.3.5.4/igb-5.3.5.4.tar.gz?r=https%3A%2F%2Fsourceforge.net%2Fprojects%2Fe1000%2Ffiles%2Figb%2520stable%2F5.3.5.4%2F&ts=1482880840&use_mirror=pilotfiber"
```

Comment:
>http://downloads.sou [](start = 43, length = 20)

Use the official source: [https://downloadmirror.intel.com/13663/eng/igb-5.3.5.4.tar.gz](https://downloadmirror.intel.com/13663/eng/igb-5.3.5.4.tar.gz) ?
Or backup it to Azure Blob Storage and comment here with a sha1sum?